engine/extension/src/extension.cpp

```diff
@@ -25,7 +26,7 @@ extern "C" {
 typedef struct ExtensionDesc
 {
     const struct ExtensionDesc* m_Next;
-    const char*                 m_Name;
+    char                        m_Name[16];
```

Comment:
It was a bit awkward to pass in a string from C# to C, and I didn't want to hold that pointer unnecessarily.
So opted for copying the name instead.